docker/integrations/README.md

```diff
@@ -0,0 +1,74 @@
+# Wazuh integrations
+
+This folder contains a docker environment with all the necessary to test integrations with Splunk and Elasticsearch, from the Wazuh Indexer as well as from the Wazuh manager.
+
+## Docker compose content:
+
+- 1 Splunk Indexer 9.0.4
+- 1 Wazuh stack (indexer, dashboard and manager). In the manager container there is also a Splunk Forwarder and a Logstash in the `/opt` folder
+- 1 Elastic stack (Elasticsearch,Kibana and the setup container)
+- 1 Opensearch stack (Opensearch and Opensearch dashboards)
+- 1 Logstash 8.6.2
+- 1 Generator that will automatically generate all the required certificates and download the required packages
+
+## Additional content:
+
+- Dashboards for Splunk and Kibana
+- Sample alerts for the last 7 days after starting the environments. Those are inside the wazuh-manager in `/var/ossec/logs/alerts/sample_alerts.json` and also in the `alerts.json` file merged with the non-sample data.
+
+## Requirement:
+
+- Internet connection
+- Docker
+- Docker compose
+
+## Usage
+
+In the .env file it is possible to configure the desired version of the Wazuh stack. It will only work with already released versions.
+
+After that, running `docker compose up -d` will start all the containers. Once the start process finishes, the integrations will be configured. It is necessary to manually start the Splunk integration from manager by running `/opt/splunkforwarder/bin/splunk start --accept-license` in the Wazuh manager container.To stop the environment and cleare it, use `docker compose down`.
```

Comment:
There's a typo in "cleare".

Reply:
Fixed [7f77da9](https://github.com/wazuh/wazuh-kibana-app/pull/5308/commits/7f77da90dc8c3e59562e58cdb6adf825c10f2e7d)